### ISSUE TITLE
Big code blocks break the editor

### ISSUE THREAD

**Issue by spacekookie** [BookStackApp/BookStack#803]
### For Bug Reports

* BookStack Version: `BookStack v0.19.0`
* PHP Version: `7.0.22`
* MySQL Version: `Ver 14.14 Distrib 5.7.21`

##### Current Behavior

When creating a block in the editor that is too long, the preview window will break the editor in a way that makes it very difficult to fix. See attached screenshot

https://imgur.com/a/YYWGf

##### Steps to Reproduce

Copy this into the editor:

```
    If you're new to eating Thai food, this vegan peanut sauce can be a great introduction into the delicious cuisine of Thailand, since it is easy to make and not spicy. Thai cuisine is light, healthy, fragrant and full of vegetables. While it can have a fiery kick, it doesn't have to. If you're ordering out, be sure to specify you would like your dish mild, at least until you have a sense for the amount of spiciness you can handle.
```

**Comment by ssddanbrown** [comment]
Hi @spacekookie,

Can you try updating to the latest version of BookStack and let me know if it's still an issue?

I think this was fixed under #715 and #716.

**Comment by tmikaeld** [comment]
I can verify that this is fixed in 0.21.

Note: 

If you're trying to upgrade and have added your own files, make sure to ignore them in git.
First: `git checkout public/robots.txt`
Then: `git pull origin release && composer install && php artisan migrate`

**Comment by ssddanbrown** [comment]
Thanks @tmikaeld for confirming, Will go ahead and close this.

If it remains to be an issue please feel free to continue to comment here and I'll re-open.
